kbase-extension/static/kbase/js/common/error.js

```diff
@@ -78,9 +80,38 @@ define([
         }
     }
 
+    function reportCellError(title, preamble, error) {
+        const t = html.tag,
+            div = t('div'),
+            p = t('p');
```

Comment:
small thing but could you add more descriptive names for p and t? single letter variable names make it a bit hard to parse what's going on

Reply:
I can change `t` (which I guess should be `tag`), but that's supposed to be a generator for DOM elements. That whole library is effectively a wrapper around HTML. It's used further down like:
```
p('some text')
```
which would generate
```
<p>some text</p>
```

I would normally agree that single-letter names are harder to read than not, but these are intended to be standins for DOM elements.